superset-frontend/src/views/CRUD/data/database/DatabaseModal/index.test.tsx

```diff
@@ -1198,6 +1198,41 @@ describe('DatabaseModal', () => {
       });
 
       describe('SSH Tunnel Form interaction', () => {
+        test('properly interacts with SSH Tunnel form textboxes for dynamic form', async () => {
```

Comment:
It seems the CI it's failing:
```
Functional: Create new database › SQL Alchemy form flow › SSH Tunnel Form interaction › properly interacts with SSH Tunnel form textboxes for dynamic form

    TestingLibraryElementError: Unable to find an element with the text: /step 2 of 2/i. This could be because the text is broken up by multiple elements. In this case, you can provide a function for your text matcher to make your matcher more flexible.
```

More likely because you need the `allow_ssh_tunneling` set to `true` for Postgres engine, otherwise it wont render the form.